src/JetMaker.cc

```diff
@@ -521,7 +521,9 @@ void JetMaker::operator()(edm::Event& evt, vlq::JetCollection& jets) {
             jet.setNConsts      ( (h_jetcMultip.product())->at(ijet) + (h_jetnMultip.product())->at(ijet) ) ; 
             jet.setGroomedMassCorr (masssmear * massCorr * (1 + jecShift_*unc) ) ; 
             jet.setNSubjets (nsubjets) ; 
-            jet.setNSubjetsBTaggedCSVL (nsubjetsbtaggedcsvl) ; 
+            jet.setNSubjetsBTaggedCSVL (nsubjetsbtaggedcsvl) ;
+            if (ijet == 0) jet.setIsleading(true);
+            else if (ijet == 1) jet.setIs2ndleading(true) ;
```

Comment:
I need to understand why this is needed

Reply:
Its purpose was to be able to check in VLQAna.cc, whether a jet in one of the vectors of 'goodTaggedJets' such as the 'goodTopTaggedJets' was the leading AK8 jet. That was the easiest way in which I could think of doing it, though there may be problems in how it is implemented